examples/src/bin/aes.rs

```diff
@@ -73,14 +73,14 @@ fn main() -> ! {
     );
     let sw_decrypted = block;
 
-    assert!(eq(&sw_encrypted.into(), &hw_encrypted));
-    assert!(eq(&sw_decrypted.into(), &hw_decrypted));
+    assert!(eq(&sw_encrypted, &hw_encrypted));
```

Comment:
You probably won't need the `eq` - you can do it like this `assert!(&sw_encrypted as &[u8] == &hw_encrypted);`